ludwig/backend/ray.py

```diff
@@ -814,6 +817,36 @@ def initialize(self):
         # Disable placement groups on dask
         dask.config.set(annotations={"ray_remote_args": {"placement_group": None}})
```

Comment:
How does this setting interact with the new logic below?

Reply:
When we set a placement group as a dask annotation it overrides the global settings and results in all dask computations going through the placement group. That's why the scope is limited to only the preprocessing step. At initialization we disable all placement groups so that dask has access to all the resources in the cluster. This was causing some issues with hyperopt earlier as far as i understand.